### ISSUE TITLE
Single instance for rpc and webRpc

### ISSUE THREAD

**Issue by ykolomiets** [metarhia/impress#854]
Hello, guys.
I have a question again. 
Can we use a single instance for RPC and WebRPC simultaneously?

**Comment by nechaido** [comment]
@ykolomiets no, the only option to have two RPC servers is to have worker instance for each of them.

**Comment by tshemsedinov** [comment]
That's Impress clustering strategy

**Comment by ykolomiets** [comment]
Could any hacks help us?

**Comment by nechaido** [comment]
@ykolomiets if you provide problem description, I might be able to help you.

**Comment by nechaido** [comment]
{
   webRpc: {
    protocol:  'jstp',
    transport: 'ws',
    address:   '*',
    ports:     [8000],
    applications: ['example'],
    slowTime:  '1s'
  },
}

**Comment by tshemsedinov** [comment]
@ykolomiets you can exchange events between instances. If I remember rightly we have automatic event retranslation.

**Comment by ykolomiets** [comment]
Example?

**Comment by belochub** [comment]
@ykolomiets, take a look at these examples:
https://github.com/metarhia/impress/blob/master/applications/example/init/globalEvents.renameToJs
https://github.com/metarhia/impress/blob/master/applications/example/init/event.renameToJs
https://github.com/metarhia/impress/blob/master/applications/example/init/clusterTest.renameToJs

Also, there are more methods available, like `application.sendEventToNode` for example, that can be used to send an event to an exact instance instead of broadcasting it. You can check them out here:
https://github.com/metarhia/impress/blob/master/lib/cloud.js

**Comment by ykolomiets** [comment]
@belochub, Okay. I have tried this. Whit `application.sendEventToNode` I found some problems, maybe it's your bug in function `setupEventHandlers` or `emitRemoteEvent`. But with some corrections, I did it and sent an event with several parameters from one worker to another. But I want to pass callback also, so path of client call is `client => webrpc api (S1N2) => (S1N1) => processing => callback to client`
